packages/react-components/src/components/attribution/verified-badge.tsx

```diff
@@ -1,12 +1,12 @@
-import { cx } from 'emotion'
 import React from 'react'
 type Props = {
     className?: string
     size?: number
     fill?: string
 }
+
 const VerifiedBadge = ({ className = '', size = 17, fill = '#15CDFF' }: Props) => (
-    <svg className={cx(VerifiedBadge.className, className)} height={size} width="19px" viewBox="0 0 19 17">
+    <svg className={[VerifiedBadge.className, className].join(' ')} height={size} width="19px" viewBox="0 0 19 17">
```

Comment:
A lot of SDK components have a publicly available `className`. This lets us keep that and also add a custom className